ddtrace/internal/telemetry/writer.py

```diff
@@ -451,8 +451,8 @@ def _app_started_event(self):
             [
                 ("data_streams_enabled", config._data_streams_enabled, "unknown"),
                 ("appsec_enabled", config._appsec_enabled, "unknown"),
-                ("trace_propagation_style_inject", str(config._propagation_style_inject), "unknown"),
-                ("trace_propagation_style_extract", str(config._propagation_style_extract), "unknown"),
+                ("DD_TRACE_PROPAGATION_STYLE_INJECT", str(",".join(config._propagation_style_inject)), "unknown"),
+                ("DD_TRACE_PROPAGATION_STYLE_EXTRACT", str(",".join(config._propagation_style_extract)), "unknown"),
```

Comment:
Do we need to make this change? The current configuration name should be supported by the backend